src/components/celestialmap/skymap_config.js

```diff
@@ -0,0 +1,304 @@
+            s0: { shape: "ellipse", fill: "#ff0000", opacity: 0.3 },                                // Lenticular galaxy
+            sd: { shape: "ellipse", fill: "#ff0000", opacity: 0.3 },                                // Dwarf galaxy
+            e: { shape: "ellipse", fill: "#ff0000", opacity: 0.3 },                                // Elliptical galaxy
+            i: { shape: "ellipse", fill: "#ff0000", opacity: 0.3 },                                // Irregular galaxy
+            oc: { shape: "circle", fill: "#ffcc00", stroke: "#ffcc00", width: 1.0 },  // Open cluster
+            gc: { shape: "circle", fill: "#ff9900", opacity: 0.4 },                                 // Globular cluster
+            en: { shape: "square", fill: "#ff00cc" },                                 // Emission nebula
+            bn: { shape: "square", fill: "#ff00cc", stroke: "#ff00cc", width: 2 },    // Generic bright nebula
+            sfr: { shape: "square", fill: "#cc00ff", stroke: "#cc00ff", width: 2 },    // Star forming region
+            rn: { shape: "square", fill: "#00ooff", opacity: 0.3 },                                 // Reflection nebula
+            pn: { shape: "diamond", fill: "#00cccc", opacity: 0.5 },                                // Planetary nebula
+            snr: { shape: "diamond", fill: "#ff00cc", opacity: 0.3 },                                // Supernova remnant
+            dn: { shape: "square", fill: "#999999", stroke: "#999999", width: 2 },    // Dark nebula grey
+            pos: { shape: "marker", fill: "#cccccc", stroke: "#cccccc", width: 1.5 }   // Generic marker
+        }
+    },
+    constellations: {
+        show: false,    // Show constellations
+        names: false,   // Show constellation names
+        namesType: false,   // Show short constellation names (3 letter designations)
+        nameStyle: {
+            fill: "#cccc99", align: "center", baseline: "middle", opacity: 0.8,
+            font: ["bold 14px Helvetica, Arial, sans-serif",  // Different fonts for brighter &
+                "bold 12px Helvetica, Arial, sans-serif",  // sdarker constellations
+                "bold 11px Helvetica, Arial, sans-serif"]
+        },
+        lines: false,   // Show constellation lines
+        lineStyle: { stroke: "#cccccc", width: 1, opacity: 0.6 },
+        bounds: false,  // Show constellation boundaries
+        boundStyle: { stroke: "#cccc00", width: 0.5, opacity: 0.8, dash: [2, 4] }
+    },
+    planets: {
+        show: true,
+        which: ["sol", "mer", "ven", "ter", "lun", "mar", "jup", "sat", "ura", "nep"],
+        // Font styles for planetary symbols
+        symbolStyle: {
+            fill: "#00ccff", font: "bold 17px 'Lucida Sans Unicode', Consolas, sans-serif",
+            align: "center", baseline: "middle"
+        },
+        symbolType: "disk",
+        symbols: {  // Character and color for each symbol in 'which', simple circle \u25cf
```

Comment:
Might be personal preference/something to ask Wayne, but shortened planet names vs. full names (especially when some named stars are fully written out)?

Reply:
Good question. This is a decision made by the library and not me, but I'm sure there's a way we could override it if we wanted to change the name display.